<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="curve">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_merge">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="channel_0">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>shg</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="channel_1">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>shg</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="channel_2">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>shg</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R通道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>G通道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B通道</string>
    </property>
   </widget>
   <widget class="QComboBox" name="folder">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
